style.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,600;1,200;1,600&display=swap');
+
+body {
+  height: 100vh;
+  background-image: url(./images/headerbg.svg);
+  font-family: "Poppins", sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+aside a {
+  margin-left: 30px;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+}
+
+.logo a {
+  height: 20px;
+  width: 76px;
+  top: 10px;
+  left: 12px;
+  font-weight: 700;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+aside {
+  margin-left: 30px;
+}
+
+.container {
+  position: absolute;
+  top: 30%;
+  left: 15%;
+  right: 15%;
+  align-items: center;
+  margin: auto;
+}
+
+header {
+  font-size: 42px;
+  font-weight: 700;
+  height: 122px;
+  width: 920px;
+}
+
+.description {
+  height: 98px;
+  width: 920px;
+  top: 134px;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: left;
+  position: relative;
+}
+
+.socials h6 {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #7f8cff;
+}
+
+.socials {
+  position: static;
+  width: 150px;
+  height: 24px;
+}
+
+.socials i {
+  color: #505f79;
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    background-image: url(./images/headerbg.svg);
```

Comment:
- [ ] Kindly ensure the background image is properly positioned as it is in the original design. You can also remove it from the body element and put it in the appropriate section
- [ ] Also, please try to adjust your screen to remove horizontal scrolling as it does not give a good user experience